lib/pgBackRest/Protocol/Storage/Helper.pm

```diff
@@ -146,85 +96,18 @@ sub storageRepo
             {name => 'strStanza', optional => true, trace => true},
         );
 
-    if (!defined($strStanza))
-    {
-        if (cfgOptionValid(CFGOPT_STANZA) && cfgOptionTest(CFGOPT_STANZA))
-        {
-            $strStanza = cfgOption(CFGOPT_STANZA);
-        }
-        else
-        {
-            $strStanza = STORAGE_REPO;
-        }
-    }
```

Comment:
So the repo objects are no longer based on the stanza? How does this now work when we say, load a backup.info file with infoBackupNewLoad(storageRepo(), STRDEF(STORAGE_REPO_BACKUP "/" INFO_BACKUP_FILE), and we have multiple stanzas? How does it determine the correct path?

Reply:
Scratch this - I forgot storage/helper.c initializes the stanza via storageRepo/Spool functions.

Reply:
Yep.